rfc-parser/src/main/java/tech/pantheon/yanginator/parser/services/RFCStringUtil.java

```diff
@@ -48,41 +49,77 @@ public static String splitTokens(String line) {
         line = line.replaceAll("\\) ", " ) ");
         line = line.replaceAll(" \\[", " [ ");
         line = line.replaceAll("] ", " ] ");
-        line = line.replaceAll("\"}\"","\"}\" )");
-        line = line.replaceAll("\"\\{\""," ( \"{\"");
-        line = line.replaceAll(CLOSED_BRACKET.getCharacterValue(), "] ?");
+        line = line.replaceAll("\"}\"", "\"}\" )");
+        line = line.replaceAll("\"\\{\"", " ( \"{\"");
+        line = line.replaceAll(RIGHT_BRACKET.getCharacterValue(), "] ?");
 
         return line;
     }
 
     public static String tryReplaceTokens(final String word) {
-        if(word.equals(DOUBLE_QUOTE.getStringValue())){ return DOUBLE_QUOTE.getBnfValue();}
-        if(word.equals(SINGLE_QUOTE.getStringValue())){ return SINGLE_QUOTE.getBnfValue();}
-        if(word.equals(SEMICOLON.getStringValue())){ return SEMICOLON.getBnfValue();}
-        if(word.equals(COLON.getStringValue())){ return COLON.getBnfValue();}
-        if(word.equals(LEFT_BRACE.getStringValue())){ return LEFT_BRACE.getBnfValue();}
-        if(word.equals(RIGHT_BRACE.getStringValue())){ return RIGHT_BRACE.getBnfValue();}
-        if(word.equals(DOT.getStringValue())){ return DOT.getBnfValue();}
-        if(word.equals(DASH.getStringValue())){ return DASH.getBnfValue();}
-        if(word.equals(UNDERSCORE.getStringValue())){ return UNDERSCORE.getBnfValue();}
-        if(word.equals(EQUAL.getStringValue())){ return EQUAL.getBnfValue();}
-        if(word.equals(LEFT_PARENTHESIS.getStringValue())){ return LEFT_PARENTHESIS.getBnfValue();}
-        if(word.equals(RIGHT_PARENTHESIS.getStringValue())){ return RIGHT_PARENTHESIS.getBnfValue();}
-        if(word.equals(OPEN_BRACKET.getStringValue())){ return OPEN_BRACKET.getBnfValue();}
-        if(word.equals(CLOSED_BRACKET.getStringValue())){ return CLOSED_BRACKET.getBnfValue();}
-        if(word.equals(FORWARD_SLASH.getStringValue())){ return FORWARD_SLASH.getBnfValue();}
-        if(word.equals(PIPE.getStringValue())){ return PIPE.getBnfValue();}
-        if(word.equals(ZERO.getStringValue())){ return ZERO.getBnfValue();}
-        if(word.equals(DOUBLE_DOT.getStringValue())){ return DOUBLE_DOT.getBnfValue();}
+        if (word.equals(DOUBLE_QUOTE.getStringValue())) {
```

Comment:
there are too many "if" conditions here. we should consider constructing a MAP for this.

Reply:
I noticed that this is from the old parser which will be removed. consider irrelevant if this will not be used later.

Reply:
This class is deprecated, it is currently not used.